pyproject.toml

```diff
@@ -26,9 +26,7 @@ importlib-resources = "^6.4.0"
 exasol-bucketfs = ">=0.6.0,<1.0.0"
 click = "^8.0.4"
 pyexasol = ">=0.25.0,<1.0.0"
-pandas = "^1.1.0"
-# request from Mikhail:
-# pandas = "^2.2.3"
+pandas = "^2.2.3"
 pydantic = "^1.10.2"
```

Comment:
We should upgrade pydantic to version 2 it might otherwise conflict with other projects

Reply:
See ticket
* #233